tests/snapshot-test/next/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  root: true,
```

Comment:
If this field is not specified, Lint will run with the configuration merged with [`.eslintrc.js` in the root directory](https://github.com/moneyforward/eslint-config-moneyforward/blob/main/.eslintrc.js).